presto-jdbc/src/main/java/io/prestosql/jdbc/PrestoDatabaseMetaData.java

```diff
@@ -1410,6 +1410,12 @@ public boolean isWrapperFor(Class<?> iface)
         return iface.isInstance(this);
     }
 
+    ResultSet getDescribeOutput(String statementName)
+            throws SQLException
+    {
+        return select("DESCRIBE OUTPUT " + statementName);
```

Comment:
There is no need for `PrestoPreparedStatement` to use `PrestoDatabaseMetaData`, so you can remove this method.